Lib/test/test_inspect.py

```diff
@@ -43,8 +43,6 @@
 from test import inspect_stringized_annotations
 from test import inspect_stringized_annotations_2
 
-from test.test_import import _ready_to_import
```

Comment:
Oh no, we should not make tests inter-dependents. This change goes to the right direction!